packages/@glimmer/opcode-compiler/lib/wrapped-component.ts

```diff
@@ -22,17 +22,27 @@ import { EMPTY_ARRAY } from '@glimmer/util';
 export class WrappedBuilder<Locator> implements CompilableProgram {
   public symbolTable: ProgramSymbolTable;
   private compiled: Option<number> = null;
+  private attrsBlockNumber: number;
 
   constructor(
     private compiler: Compiler<OpcodeBuilder<Locator>>,
     private layout: LayoutWithContext<Locator>
   ) {
     let { block } = layout;
 
+    let symbols = block.symbols.slice();
     this.symbolTable = {
```

Comment:
Can you move this assignment to L45 just so it's easier to understand that you are going to mutate `symbols` and place it in the `symbolTable`.

Reply:
Yep, makes sense. Will update.

Reply:
Done! Thanks for reviewing!